HTML/HTML (Plain).sublime-syntax

```diff
@@ -88,7 +88,8 @@ variables:
   tag_name_break_char: '[{{ascii_space}}/<>]'
   tag_name_break: (?={{tag_name_break_char}})
   tag_name_char: '[^{{tag_name_break_char}}]'
-  tag_name: '[A-Za-z]{{tag_name_char}}*'
+  tag_name_start: '[A-Za-z]'
```

Comment:
How about adding a comment here that explains the intent of this variable, i.e. being used for extensions? Will make it more obvious in the future that this is something external syntaxes (or internal even) actually depend on and reduces the likelyhood of changing/removing it by accident.

Reply:
Added a note and some docs at the beginning of the HTML (Plain) syntax.